apps/api/src/bootstrap.ts

```diff
@@ -114,9 +126,20 @@ export async function bootstrap(expressApp?): Promise<INestApplication> {
     .addTag('Organizations')
     .addTag('Execution Details')
     .build();
-  const document = SwaggerModule.createDocument(app, options);
-
-  SwaggerModule.setup('api', app, document);
+  const document = injectDocumentComponents(SwaggerModule.createDocument(app, options));
+
+  SwaggerModule.setup('api', app, {
+    ...document,
+    info: {
+      ...document.info,
+      title: `DEPRECATED: ${document.info.title}. Use /openapi.{json,yaml} instead.`,
+    },
+  });
+  SwaggerModule.setup('openapi', app, document, {
+    jsonDocumentUrl: 'openapi.json',
+    yamlDocumentUrl: 'openapi.yaml',
+    explorer: process.env.NODE_ENV !== 'production',
+  });
```

Comment:
We can keep both endpoints up, and remove the deprecated endpoint early next year in case customers are using it to codegen clients. I can raise a ticket to remove the deprecated endpoint if we are happy with this direction.